README.md

```diff
@@ -1 +1,11 @@
 # hawify
```

Comment:
Nice job! Maybe add information about how to start your project if someone clones it?